### PR TITLE
New Version Numbering

### PR DESCRIPTION
**Pull request by souldanger**
updated Marquee
News Post
added CentOS to RHEL
various updates to macOS
added hiding index.html to .htaccess
added redirect to .htaccess

### CODE REVIEW

File: downloads/linux.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ current version 2017-10
 \\
 system requirements   
 &#8226; 64-bit Ubuntu 14.04+, Debian 8+, Mint 17+   
-&#8226; openSUSE Leap 42.2+, Tumbleweed   
+&#8226; openSUSE Leap 42.2+, Tumbleweed
```
Comment (jengelh) [inline downloads/linux.md]: Was sollen diese unnützen(?) Spaces? Kann das bitte unterlassen werden?

File: downloads/windows.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Running Iridium portable version works out of the box. As a recommendation we re
 If you are looking for previous versions, please check out the [Builds Archive](https://downloads.iridiumbrowser.de/windows/ "go to Builds Archive"){:target="_blank"}<br/>
 \\
 system requirements   
-&#8226; Windows 7+   
+&#8226; Windows 7 or later
```
Comment (jengelh) [inline downloads/windows.md]: You can't have one place say "macOS 10.9+" and another "7 or later". You need to decide for one style.

Comment (souldanger) [reply downloads/windows.md]: agreed & fixed!

File: downloads/linux.md (hunk 1 above)
Comment (souldanger) [reply downloads/linux.md]: Das sind Zeilenumbrüche...

**Pull request merged**